local-pv-provisioner/controllers/persistent_volume.go

```diff
@@ -0,0 +1,58 @@
+package controllers
+
+import (
+	"context"
+	"crypto/sha1"
+	"encoding/hex"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const (
+	storageClass     = "local-storage"
```

Comment:
This storage class name is quite important and an external specification
to bind the persistent volumes created by `local-pv-provisioner` through
persistent volume claims (PVC).

I'd like to export this as a public constant.

Reply:
Changed to public constant.